kotlinx.fuzz.engine/src/main/kotlin/kotlinx/fuzz/reproduction/ReproducerTestTemplate.kt

```diff
@@ -0,0 +1,12 @@
+package kotlinx.fuzz.reproduction
+
+import com.squareup.kotlinpoet.CodeBlock
+
+interface ReproducerTestTemplate {
```

Comment:
Again, the name of this interface does not tell me what it does. And in this case I truly don't understand it yet, so it is hard for me to give suggestions. I think we should add documentation here and then think about new name

Reply:
Added documentation. Renamed to `TestReproducerTemplate`, it should be more clear